create-rust-app/src/auth/permissions/mod.rs

```diff
@@ -45,6 +50,11 @@ impl Role {
     /// assigns every role in `roles` to the User whose id is [`user_id`](`ID`)
     ///
     /// returns true if successful
+    ///
+    /// # Errors
+    /// * infallible
+    ///
+    /// TODO: don't return a result if we never fail, or return a result and not a bool
     pub fn assign_many(db: &mut Connection, user_id: ID, roles: Vec<String>) -> Result<bool> {
```

Comment:
regarding these todo statements, I think that returning a `Result<()>` is more ergonomic for the user than `bool`, just because then the function signiture explicitly tells users that the returned value indicates success/failure rather than having them need to read the documentation to know that.

Reply:
resolved by #372